assets/javascripts/feature.js

```diff
@@ -1,79 +1,125 @@
-// jshint esversion: 6
+var disablePernametly = false;
```

Comment:
```suggestion
var disablePermanently = false;
```

Reply:
It doesn't look resolved, I still see `Pernamently`.

Reply:
i hate words with *manent* ...